js-foundations.js

```diff
@@ -3,28 +3,62 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
-
+function isDivisbleByThree(num){
+    let rem = num % 3;
+    if(rem === 0 || num === 0){
+        return num + " is divisble by three";
+    }
+    return num + " is not divisble by three";
+}
+console.log(isDivisbleByThree(15));
+console.log(isDivisbleByThree(0));
+console.log(isDivisbleByThree(385));
+console.log(isDivisbleByThree(-7));
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
-
+let helloMe = {
+    firstName: "Ryan",
+    lastName: "Young",
+    height: "5 foot 11",
+    weight: "160 pounds",
+    whoIsMe: function(){
+        return "My name is " + this.firstName + " " + this.lastName + " and I am " + this.height +" and " + this.weight
+    }
+}
+console.log(helloMe.whoIsMe());
 
```

Comment:
Looks really great. You can use string interpolation here to make your code a bit cleaner.